src/patternfly/components/Page/page.scss

```diff
@@ -482,12 +480,21 @@
     padding: 0;
```

Comment:
You shouldn't need this

Reply:
You still shouldn't need this. It gets built using the breakpoint loop below with "base" as the value in the map. The compiled CSS currently looks like:

```
.pf-c-page__main-section.pf-m-no-padding {
  padding: 0;
}

.pf-c-page__main-section.pf-m-padding {
  padding: var(--pf-c-page__main-section--PaddingTop) var(--pf-c-page__main-section--PaddingRight) var(--pf-c-page__main-section--PaddingBottom) var(--pf-c-page__main-section--PaddingLeft);
}

.pf-c-page__main-section.pf-m-no-padding {
  padding: 0;
}

@media (min-width: 576px) {
  .pf-c-page__main-section.pf-m-padding-on-sm {
    padding: var(--pf-c-page__main-section--PaddingTop) var(--pf-c-page__main-section--PaddingRight) var(--pf-c-page__main-section--PaddingBottom) var(--pf-c-page__main-section--PaddingLeft);
  }

  .pf-c-page__main-section.pf-m-no-padding-on-sm {
    padding: 0;
  }
}
```